packages/editor-ui/src/components/Node.vue

```diff
@@ -502,7 +502,9 @@ export default defineComponent({
 		}
 	},
 	mounted() {
-		this.setSubtitle();
+		setTimeout(() => {
```

Comment:
shouldn't we do this in the `isActive` watcher as well? and if yes, wouldn't it make sense to move the `setTimeout` inside `setSubtitle` ?

Reply:
no settimeout is only needed here when a lot of nodes are mounted at the same time (pasting a big workflow for example)..

Reply:
previously `this.$emit('run')` happened after `setSubtitle` finished. are we certain that breaking this sequence isn't going to cause any issues?

Reply:
yes, it should be fine.. that's related to showing success state for nodes in executions view

Reply:
I think `this.$nextTick(() => {})` be more appropriate here.

Reply:
hmm I have run into problems in the past with this.. that's why i default to `setTimeout` it always works

Reply:
Ok, no strong opinion for it. If it works it works.

Reply:
just tested `nextTick` and it works here.. but will stick with `setTimeout` if you don't mind then